### DISCUSSION TITLE
Yi Camera with FH8632 SoC

### DISCUSSION THREAD

**Discussion by DGA45** [roleoroleo/yi-hack-Allwinner-v2#510] [Q&A]
Hello,
I bought the following camera: https://fr.aliexpress.com/item/1005004622553546.html
It is working with the Yi IOT app and returns "Xioayi" as brand.
I tried the Yi hack, but not working, despite we can recognize similar files structure in the existing logs file created by the camera at startup/wifi login.
I disassemble it and the SoC is a FullHan FH8632 one.

I tried also several alternatives of the Yi hack (test instead of Factory folder, equip_test.sh instead of factory_test.sh, the wifi/config.sh trick, etc...), but I cannot get any log written on the SD card to confirm the bash execution.
However, I am pretty sure "something" happens.
Indeed, when scripts are not there or incorrect, the camera will start by emitting some sound and recreate its log folder.
With the "factory" scripts, the camera is not emitting sounds and do not create its log folder, but I see nothing performed by the script (echoing text into the /tmp/sd/log.txt file).
I also tried /tmp/hd1 and /var/tmp/sd without success.

Maybe the /tmp/sd is mounting as RO by default ?

How could I verify that the script is executing and go further with this hack on this new cam ?

**Comment by roleoroleo** [comment]
Probably the path is not /tmp/sd
Something useful in the log?

**Reply by DGA45** [reply]
The logs are referring to /tmp/sd, but since I found other possible directories on the web, I did also try the other without success.
I join my logs here in case you would find some additional things:
[file.txt](https://github.com/roleoroleo/yi-hack-Allwinner-v2/files/9725475/file.txt)
[ps1.txt](https://github.com/roleoroleo/yi-hack-Allwinner-v2/files/9725476/ps1.txt)

During my investigations on the web, I also found some real factory scripts for similar cameras (FullHand, ANYKAV200 or QG SoC). You can have a look here, I think it will be valuable information for you: http://47.101.136.209/factory_tool/release/FactoryFile/

I will also try to analyze them and try other filenames/folders as per those scripts.

**Comment by pmariett** [comment]
Hi,
  I have cameras that can be used via the Yi app but with an FH8632 shipset.
I tried several manipulations to launch commands via the sd card but without success.
If anyone has any leads...

**Comment by DGA45** [comment]
Hi,
I succeeded in having root access and launching a recompiled rstp server.
Somebody else seems interested, I propose to continue on https://github.com/roleoroleo/yi-hack-Allwinner-v2/issues/528

**Comment by roleoroleo** [comment]
Could you send me a link to buy this item (aliexpress, amazon, etc...)?

**Reply by redphx** [reply]
I bought mine (Feb 17) from their official Tmall store: https://detail.tmall.com/item_o.htm?id=559840219176&skuId=4916181012549

**Reply by DGA45** [reply]
Mine on Aliexpress:
https://fr.aliexpress.com/item/1005004622553546.html?spm=a2g0o.order_list.order_list_main.43.29bc5e5bWqtmzH&gatewayAdapt=glo2fra

**Reply by pmariett** [reply]
https://fr.aliexpress.com/item/1005004494042098.html?spm=a2g0o.order_list.order_list_main.16.39c55e5bw3UcfY&gatewayAdapt=glo2fra for me
